foursight_core/react/api/encryption.py

```diff
@@ -0,0 +1,111 @@
+# Encryption utility class.
```

Comment:
Resurrected this for experimental /accounts page support. The optional accounts.json is encrypted (manually, locally) using ENCODED_AUTH0_SECRET value from GAC and decrypted on the fly by the API code using this secret value.

Reply:
I'm pondering whether I think that's adequate to keep it secret on a public site. @willronchetti what's your thought on this?